### PR TITLE
add autoRender option

### PR DESCRIPTION
**Pull request by lcxfs1991**
In some cases, we will wanna use `return await ctx.render()`. So I add an extra option to resolve it.

### CODE REVIEW

**Comment by int64ago** [comment]
LGTM, could you please add some tests?

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "koa-views",
-  "version": "6.1.5",
+  "name": "@vemo/koa-views",
+  "version": "6.1.6",
```
Comment (int64ago) [inline package.json]: ?

File: src/index.js [hunk 2]
```diff
@@ -11,13 +11,14 @@ module.exports = viewsMiddleware
...
   path,
-  { engineSource = consolidate, extension = 'html', options = {}, map } = {}
+  { autoRender = true, engineSource = consolidate, extension = 'html', options = {}, map } = {}
 ) {
   return function views(ctx, next) {
     if (ctx.render) return next()
 
     ctx.response.render = ctx.render = function(relPath, locals = {}) {
-      return getPaths(path, relPath, extension).then(paths => {
+      let prefixPath = isAbsolute(relPath) ? '' : path
```
Comment (int64ago) [inline src/index.js]: One PR does one thing

**Comment by int64ago** [comment]
BTW, it could be better if you rebase all commits to one commit :-)

**Comment by lcxfs1991** [comment]
Sorry. I will fix this stuff soon.

@int64ago

**Comment by int64ago** [comment]
ping @lcxfs1991 ?

**Comment by lcxfs1991** [comment]
@int64ago fixed. Please check.

**Comment by lcxfs1991** [comment]
Please support us and star this project. Haha.

[996.icu](https://github.com/996icu/996.ICU)

**Comment by int64ago** [comment]
Well done! Thanks!

**Comment by int64ago** [comment]
Via `v6.2.0`

**Pull request merged**